### DISCUSSION TITLE
Unified Loader: Which Models for PLUS (high strength) and PLUS FACE (portraits)?

### DISCUSSION THREAD

**Discussion by PresTrembleyIIIEsq** [cubiq/ComfyUI_IPAdapter_plus#551] [Q&A]
Hello,

Everything is working fine if I use the Unified Loader and choose either the `STANDARD (medium strength)` or `VIT-G (medium strength)` presets, but I get `IPAdapter model not found` errors with either of the PLUS presets.

I could have sworn I've downloaded every model listed on the main page here. Can anyone confirm which models are related to these two PLUS presets so I can triple-check just those ones?

![Screenshot](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/140981996/2d370a19-6d13-4807-93a5-364fb53c58a9)

**Comment by cubiq** [comment]
you are probably missing the VIT-H clip vision

**Reply by PresTrembleyIIIEsq** [reply]
Thanks for the suggestion. I assume "download and rename" means the file itself should be called `CLIP-ViT-H-14-laion2B-s32B-b79K.safetensors`? I did that for both clip vision files again just to be sure (downloaded into `/ComfyUI/models/clip_vision`), and still get the same error.

If it helps, here's the error text:

```
Error occurred when executing IPAdapterUnifiedLoader:

IPAdapter model not found.

  File "[...]/ComfyUI/execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "[...]/ComfyUI/execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "[...]/ComfyUI/execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "[...]/ComfyUI/custom_nodes/ComfyUI_IPAdapter_plus/IPAdapterPlus.py", line 468, in load_models
    raise Exception("IPAdapter model not found.")
```

**Reply by garlinkchen-pixel** [reply]
same error, but maybe u can try VIT-G（medium strength）

**Comment by yourbff** [comment]
@PresTrembleyIIIEsq I just had this issue (with the files in place and named appropriately) and was able to resolve it by uninstalling and reinstalling IPAdapter. Hope this helps!

**Comment by yym68686** [comment]
The file is placed correctly and named correctly, but the same error is still reported after uninstalling and reinstalling IPAdapter: `raise Exception("IPAdapter model not found.")`

**Reply by Jordan-Read** [reply]
did you ever find out why? I am having the same issue.

**Reply by yym68686** [reply]
Hello, I've already solved it. I downloaded a few IPAdapter type models in the model manager and that resolved the issue.

**Reply by Jordan-Read** [reply]
Yes! Thank you! The IpAdapter models did the trick!

**Reply by chnisar515** [reply]
i am getting same error so can you tell me which type of models i need to download ? Thank you.

**Reply by zhengjianfeng11** [reply]
The file name downloaded from GitHub is ip-adapter-plus-face_sdxl_vit-h.safetensors, and the file name downloaded from huggingface is ip-adapter-plus_sdxl_vit-h.safetensors. I used the file name of huggingface and it worked fine.

**Comment by airtonix** [comment]
It'd be nice if there was actual documentation on where these things are supposed to go. 
All we've got is speculation.

**Reply by airtonix** [reply]
edit here

https://github.com/cubiq/ComfyUI_IPAdapter_plus/blob/main/README.md?plain=1#L82
